tests/integration_tests/test_datatypes.py

```diff
@@ -3,13 +3,20 @@
 from guardrails.guard import Guard
 
 
-def test_passed_date_format():
-    rail_spec = """
+@pytest.mark.parametrize(
+    "date_format,date_string",
+    [
+        ("%Y-%m-%d", "2021-01-01"),  # standard date
```

Comment:
`"%Y-%m-%d"` is the default right? So the test below (not shown in the diff) also tests the cohere style date with this date format?